app/src/test/java/com/example/warriorsocial/ui/login/RegistrationCheckerTest.java

```diff
@@ -0,0 +1,29 @@
+package com.example.warriorsocial.ui.login;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class RegistrationCheckerTest {
+
+    @Test
+    public void checkIfUserLoggedIn() {
+        fail();
```

Comment:
Is this complete? Looks like a few tests are unfinished.

Reply:
New Update for dev fixes this. This was an incomplete version, opened a pull request from AlexChen_UnitTest (tested myself runs unit tests fine) to dev!